staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_discovery_test.go

```diff
@@ -529,6 +529,41 @@ func TestLegacyFallback(t *testing.T) {
 		},
 	}, doc.Items)
 }
+func TestLegacyFallback(t *testing.T) {
+	rootAPIsHandler := discovery.NewRootAPIsHandler(discovery.DefaultAddresses{DefaultAddress: "192.168.1.1"}, scheme.Codecs)
```

Comment:
nit: I know pre-existing, but does this IP address matter? Can it be empty/loopback?

Reply:
afaik it does not matter here and is specifically used for https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L1085-L1093 which is not a field we use in aggregated discovery.